.github/workflows/gobump.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: "Updates Go dependencies via gobump"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Every Sunday at 15:00
+    - cron: "0 15 * * 0"
+
+jobs:
+  update-and-push:
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+
+      - name: Update go.mod
+        run: |
+          go install github.com/lzap/gobump@latest
+          gobump -exec "go build ./pkg/..." -exec "go test ./pkg/osbuild" 2>&1 | tee gobump.log
```

Comment:
Why are we only building `./pkg` and testing `./osbuild`?  Is it too much to run with `./...`?  I suppose everything in `./internal` is covered by imports from `./pkg`, but there's a lot that would be missed by an update breaking something in `./cmd`.

Reply:
No particular reason other than time and output size, the thing is packages are vendored and I believe `./...` is testing or building all directories including `vendor/`? So let me add more explicit directories there.

I am bit worried about length of the output tho as it goes as PR input, particularly `go test` leaves ton of output behind which cannot be turned off. Let's see how it looks like first and then I can fine-tune it. If this works I will probably add a markdown output to the tool so we get nicely formatted report of an update.

Reply:
Dependencies aren't vendored in this repository.

Reply:
Ah good point changed both to `./...` then there is plenty of time on Sunday midnight on CICD so we can make sure everything tests correctly.